wolfssl/openssl/rsa.h

```diff
@@ -116,10 +116,14 @@ WOLFSSL_API int wolfSSL_RSA_sign(int type, const unsigned char* m,
                                unsigned int* sigLen, WOLFSSL_RSA*);
 WOLFSSL_API int wolfSSL_RSA_sign_ex(int type, const unsigned char* m,
                                unsigned int mLen, unsigned char* sigRet,
-                               unsigned int* sigLen, WOLFSSL_RSA*, int);
+                               unsigned int* sigLen, WOLFSSL_RSA*, int, int);
 WOLFSSL_API int wolfSSL_RSA_verify(int type, const unsigned char* m,
                                unsigned int mLen, const unsigned char* sig,
                                unsigned int sigLen, WOLFSSL_RSA*);
+WOLFSSL_API int wolfSSL_RSA_verify_ex(int type, const unsigned char* m,
+                               unsigned int mLen, const unsigned char* sig,
+                               unsigned int sigLen, WOLFSSL_RSA* rsa,
+                               int padding);
```

Comment:
I am still concerned about breaking compatibility with these _ex API's, since it was in the last v4.5.0 release. I know we've seen support tickets about the use of these API's, so I don't think customers will be happy. You may have to add another version of the API's perhaps even more generic to cover all future cases (you can use _ex2 if you want). Also the documentation for this API is missing.

Reply:
Could it be called `wolfSSL_RSA_verify_generic_padding` for example?